lib/topics/index-spec.ts

```diff
@@ -0,0 +1,161 @@
+import * as Promise from 'bluebird';
+import { assert } from 'chai';
+import * as sinon from 'sinon';
+import mediator from '../mediator';
+import Topics, {IErrorWithDataId} from './index';
+
+describe('Topics', function () {
+  let topics: Topics;
+  beforeEach(function() {
+    topics = new Topics(mediator).withPrefix('wfm:cloud').withEntity('user');
```

Comment:
[Optional] I saw that we using constants for prefixes in some modules. Maybe it's worth to use it here?